app/ide-desktop/lib/dashboard/src/authentication/src/components/app.tsx

```diff
@@ -74,14 +76,26 @@ export const SET_USERNAME_PATH = '/set-username'
 // === App ===
 // ===========
 
-/** Global configuration for the `App` component. */
-export interface AppProps {
-    /** Logger to use for logging. */
+interface BaseAppProps {
     logger: loggerProvider.Logger
     platform: platformModule.Platform
+    /** Whether the dashboard should be rendered. */
+    enableDashboard: boolean
```

Comment:
minor thing, but I think `showDashboard` would be a better name. `enableDashboard` makes me confused what "enabling" it really means, while, according to docs, this is only about showing it.